Mathlib/GroupTheory/Perm/Cycle/Basic.lean

```diff
@@ -1978,3 +2071,213 @@ theorem _root_.Finset.sum_mul_sum_eq_sum_perm (hσ : σ.IsCycleOn s) (f g : ι 
 #align finset.sum_mul_sum_eq_sum_perm Finset.sum_mul_sum_eq_sum_perm
 
 end Finset
+
+theorem subtypePerm_apply_pow_of_mem (g : Perm α) (s : Finset α)
```

Comment:
Here there a bunch of variables that can be implicit.

Reply:
Yes. And the next one as well.